shared/loadgenerator/loadgenerator.go

```diff
@@ -55,14 +57,18 @@ type GeneratorOptions struct {
 	// RequestTimeout is the maximum time waiting for a response.
 	RequestTimeout time.Duration
 	// QPS is the total QPS generated by all threads.
+	// If NumSteps is specified, for every step this value will be doubled.
 	QPS float64
 	// AllowInitialErrors specifies whether initial errors will abort the load.
 	AllowInitialErrors bool
+
+	// NumSteps controls the number of steps needed to reach full load.
+	NumSteps int
```

Comment:
should we default NumSteps to 1 if not specified?  Right now I think a test will not run if its NumSteps is not specified.

Reply:
it defaults to 1. This has been updated in addDefaults()